### PR TITLE
Babel support and Databse Contract Test

### PR DESCRIPTION
**Pull request by yiran29**
*Description of changes:*
1) AWS Remote DB User support
2) Contract Tests for MySQL and PostgreSQL
Currently have to comment out RemoteOperation field test, the issue is described in https://github.com/aws-observability/aws-otel-dotnet-instrumentation/issues/81

*Test Summary:* 
**PostgreSQL**
`===== 4 passed in 25.86s =====`

**MySQL**
`====== 4 passed in 42.45s ======`

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: test/contract-tests/tests/test/amazon/efcore/efcore_test.py
Comment (vastin) [inline test/contract-tests/tests/test/amazon/efcore/efcore_test.py:1]: Why is `local_operation` validation removed?

File: test/contract-tests/tests/test/amazon/netcore/netcore_test.py
Comment (vastin) [inline test/contract-tests/tests/test/amazon/netcore/netcore_test.py:1]: Why are `request_method` and `local_operation` validations removed?

**Comment by vastin** [comment]
There is merge conflict. Please rebase the pull request.

**Comment by vastin** [comment]
The PostgreSQL and MySQL tests take more than 10 minutes to finish. Is it possible to reduce the test duration?

**Comment by yiran29** [comment]
> The PostgreSQL and MySQL tests take more than 10 minutes to finish. Is it possible to reduce the test duration?

The time is mainly from the Auto Instrumentation Initialization

**Comment by vastin** [comment]
> > The PostgreSQL and MySQL tests take more than 10 minutes to finish. Is it possible to reduce the test duration?
> 
> The time is mainly from the Auto Instrumentation Initialization

Please add following in Dockerfile and it will eliminate the delay.
```
ENV RESOURCE_DETECTORS_ENABLED=false
```

**Comment by yiran29** [comment]
> > > The PostgreSQL and MySQL tests take more than 10 minutes to finish. Is it possible to reduce the test duration?
> > 
> > 
> > The time is mainly from the Auto Instrumentation Initialization
> 
> Please add following in Dockerfile and it will eliminate the delay.
> 
> ```
> ENV RESOURCE_DETECTORS_ENABLED=false
> ```

Updated the config and rerun the tests. Updated test summary.

File: test/contract-tests/tests/test/amazon/mysql/mysql_test.py [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+        self.assertTrue("db.operation" not in attributes_dict)
+
+    @override
+    def _assert_aws_attributes(
+        self, attributes_list: List[KeyValue], expected_span_kind: str = SPAN_KIND_CLIENT, **kwargs
+    ) -> None:
+        attributes_dict: Dict[str, AnyValue] = self._get_attributes_dict(attributes_list)
+        self._assert_str_attribute(attributes_dict, AWS_LOCAL_SERVICE, self.get_application_otel_service_name())
+        self._assert_str_attribute(attributes_dict, AWS_REMOTE_SERVICE, self.get_remote_service())
+        # TODO: https://github.com/aws-observability/aws-otel-dotnet-instrumentation/issues/81
```
Comment (vastin) [inline test/contract-tests/tests/test/amazon/mysql/mysql_test.py:140]: In [issue #81](https://github.com/aws-observability/aws-otel-dotnet-instrumentation/issues/81), there is `db.statement: SELECT * FROM Blogs`. Why is AWSRemoteOperation set to unknown?

Comment (yiran29) [reply test/contract-tests/tests/test/amazon/mysql/mysql_test.py:140]: The issue happened here: https://github.com/aws-observability/aws-otel-dotnet-instrumentation/blob/main/src/AWS.Distro.OpenTelemetry.AutoInstrumentation/AwsMetricAttributeGenerator.cs#L489
Briefly tested without regex test will show the remote operation

**Review by vastin — APPROVED** [review]
LGTM

**Pull request merged**